incubator-tools/rename_entity_type/rename_entity_type.ipynb

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.    input_bucket, input_uri_prefix = gcs_utilities.split_gcs_uri(INPUT_GCS_PATH)
</pre><p>The GCS I/O Code is very complicated, and should already be in the utilities directory</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
This split is not available in utilities , so used tool box function